profiles/features/musl/package.mask

```diff
@@ -56,7 +56,7 @@ sys-libs/glibc
 # We use eudev which is tested on both uclibc and musl
 =virtual/libudev-232-r1
 sys-apps/systemd
-sys-fs/udev
+<sys-fs/udev-249-r2
 
```

Comment:
Going to think about either keeping the mask or masking newer versions of the virtual for now to avoid changing defaults for new installs yet.